components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java

```diff
@@ -3642,6 +3646,17 @@ public boolean isConfigured() {
         return certificateManager.isConfigured();
     }
 
+    @Override
+    public CertificateMetadataDTO getCertificate(String alias) throws APIManagementException {
+        int tenantId = 0;
+        try {
+            tenantId = ServiceReferenceHolder.getInstance().getRealmService().getTenantManager()
+                    .getTenantId(tenantDomain);
+        } catch (UserStoreException e) {
+            handleException("Error while reading tenant information", e);
+        }
+        return certificateManager.getCertificate(alias, tenantId);
```

Comment:
move this line inside try

Reply:
Not possible. It's gonna give a missing return statement error